_data/providers.json

```
@@ -389,6 +389,11 @@
       "url": "https://www.volcanohosting.net/",
       "description_template": "default"
     },
+    {
+      "name": "VoltHosting",
+      "url": "https://www.volthosting.co.uk/",
+      "description": "Enable clone-remote-port (or manually set bedrock port to the Java port) so that the Bedrock's port is the Java server’s port, and connect with that port. Alternatively, request an additional port, or buy a dedicated IP address.<br>If you continue to have issues, please [contact VoltHosting support](https://volthosting.co.uk/contact.html) for further assistance."
```

l
```suggestion
      "description": "Enable clone-remote-port (or manually set bedrock port to the Java port), and connect with the Java IP and port. Alternatively, request an additional port, or buy a dedicated IP address. If you continue to have issues, please [contact VoltHosting support](https://volthosting.co.uk/contact.html) for further assistance."
```

Removed the line break, and rephrased it to be closer to the other descriptions

---

More than happy with that edit, looks good :)